ui/src/app/components/app-router.tsx

```diff
@@ -154,8 +148,3 @@ export const AppRoute = ({
     </Route>
   );
 };
-
-export const Navigate = ({ to }): React.ReactElement => {
-  const location = useLocation();
-  return <Redirect to={{ pathname: to, state: { from: location } }} />;
```

Comment:
we didn't actually use the `{ from: location }` functionality, so this can be replaced by Redirect